cmd/operator-sdk/test/local.go

```diff
@@ -206,7 +211,11 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 		testArgs = append(testArgs, strings.Split(tlConfig.goTestFlags, " ")...)
 	}
 	if tlConfig.namespace != "" || tlConfig.noSetup {
-		testArgs = append(testArgs, "-"+test.SingleNamespaceFlag, "-parallel=1")
+		// check for WATCH_NAMESPACE="" in --up-local
+		// so that operator can watch All-Namespaces in localRunMode
+		if !(tlConfig.upLocal && tlConfig.namespace == "" && cmd.Flags().Changed("namespace")) {
```

Comment:
Why the both ifs are required here? Is not possible to simplify it?

Reply:
one if is to allow `--namespace=""`, the other is to skip `singleNamespaceMode` when `--namespace=""` is set.

Reply:
i have made pr https://github.com/operator-framework/operator-sdk/pull/2149 which address these better.
we can close this pr.